HW_7/task_2.2/task.js

```diff
@@ -0,0 +1,19 @@
+class Task {
+    constructor(featureName, userStoryNumber, estimations) {
+        this.featureName = featureName;
+        this.userStoryNumber = userStoryNumber;
+        this.estimations = estimations;
+    }
+    updateTask(newData) {
+        if (newData.featureName) {
+            this.featureName = newData.featureName;
+        }
+        if (newData.userStoryNumber) {
+            this.userStoryNumber = newData.userStoryNumber;
+        }
+        if (newData.estimations) {
+            this.estimations = newData.estimations;
+        }

```

Comment:
As option you can set a default value
```suggestion
            this.featureName = newData.featureName || 'Forget to provide a name';
            this.userStoryNumber = newData.userStoryNumber || -1;
            this.estimations = newData.estimations || 'Forget to estimate';
```